components/google_cloud/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@pipedream/google_cloud",
-  "version": "0.3.7",
+  "version": "0.3.8",
```

Following [the semantic versioning guidelines](https://pipedream.com/docs/components/guidelines/#versioning), a new component should warrant a new `minor` version, not a `patch`
```suggestion
  "version": "0.4.0",
```